cherrypy/test/test_static.py

```diff
@@ -385,6 +400,17 @@ def test_unicode(self):
 
             expected = ntou('Героям Слава!', 'utf-8')
             self.assertInBody(expected)
+    
+    def ensure_unicode_filesystem(self):
```

Comment:
Based on this and the previous comment, I made the function a global function and put it in the proximity of `unicode_file`.